libcontainer/cgroups/fscommon/utils.go

```diff
@@ -129,3 +129,43 @@ func GetCgroupParamString(path, file string) (string, error) {
 
 	return strings.TrimSpace(contents), nil
 }
+
+// ParseRdmaKV: parse raw string to RdmaEntry.
+func parseRdmaKV(raw string, entry *cgroups.RdmaEntry) {
+	var value uint64
+	var err error
+
+	parts := strings.SplitN(raw, "=", 3)
+	if len(parts) == 2 {
```

Comment:
```go
if len(parts) != 2 {
    continue
```

Reply:
Following is handled in calling loop by throwing custom error.